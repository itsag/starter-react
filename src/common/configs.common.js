// App Config
export const appConfig = {
  appName: "Starter",
  developerName: "The Rift",
  developerUrl: "https://therift.dev",
};

// Supabase
export const supabaseConfig = {
  projectKey: import.meta.env.VITE_APP_SUPABASE_PROJECT_KEY,
  projectUrl: import.meta.env.VITE_APP_SUPABASE_PROJECT_URL,
};
