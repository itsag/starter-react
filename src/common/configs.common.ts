// Global Types
import type { AppConfig, SupabaseConfig } from "@/types/app.types";

// App Config
export const appConfig: AppConfig = {
  appName: "Starter",
  developerName: "The Rift",
  developerUrl: "https://therift.dev",
};

// Supabase
export const supabaseConfig: SupabaseConfig = {
  projectKey: import.meta.env.VITE_APP_SUPABASE_PROJECT_KEY,
  projectUrl: import.meta.env.VITE_APP_SUPABASE_PROJECT_URL,
};
