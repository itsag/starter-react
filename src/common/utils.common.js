// Modules
import { DateTime } from "luxon";

// Get UTC Timestamp
export const getUtcTimestamp = () => {
  return DateTime.utc().toMillis();
};
