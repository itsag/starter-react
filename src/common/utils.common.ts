// Modules
import { DateTime } from "luxon";

// Get UTC Timestamp
export const getUtcTimestamp = (): number => {
  return DateTime.utc().toMillis();
};
