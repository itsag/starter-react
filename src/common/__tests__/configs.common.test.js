// bunx jest src/common/__tests__/configs.common.test.js

import { appConfig } from "@/common/configs.common";

describe("configs.common", () => {
  describe("appConfig", () => {
    it("should match snapshot", () => {
      expect(appConfig).toMatchSnapshot();
    });
  });
});
