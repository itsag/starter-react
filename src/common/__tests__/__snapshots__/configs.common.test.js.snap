// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`configs.common > appConfig > should match snapshot 1`] = `
{
  "appName": "Starter",
  "developerName": "The Rift",
  "developerUrl": "https://therift.dev",
}
`;
